src/input/event/eventhandler_impl.cpp

```diff
@@ -1,13 +1,17 @@
 #include "eventhandler_impl.hpp"
 
-EventHandlerImpl::EventHandlerImpl()
-    : m_inputpoller(input::InputPollerFactory().create()), m_stop(false) {}
+namespace input {
+  namespace event {
```

Comment:
Same as with the other implementations, I do not think anything with `impl` in the filename/directoryname will need to be in the namespaces that are used externally (externally meaning outside of the module/submodule). Implementations are only referenced internally